pages/remote.jsx

```diff
@@ -0,0 +1,130 @@
+var React = require('react');
+var Header = require('../components/header.jsx');
+var Footer = require('../components/footer.jsx');
+var HeroUnit = require('../components/hero-unit.jsx');
+var ImageTag = require('../components/imagetag.jsx');
+var generateHelmet = require('../lib/helmet.jsx');
+
+var FullWidth = React.createClass({
```

Comment:
this component, and the other ones really, are probably good to move up as universal components instead, so that we can clean up the other pages.